packages/cli/lib/validators/constants.js

```diff
@@ -4,11 +4,13 @@ const SUCCESS = 'SUCCESS';
 
 const VALIDATION_RESULT = { WARNING, FATAL, SUCCESS };
 const VALIDATOR_KEYS = {
-  dependency: 'dependency',
-  module: 'module',
+  themeDependency: 'themeDependency',
```

Comment:
This may have already happened, but I _think_ there are some BE systems that make use of these keys. Just double checking that they've updated their services in response to this.